a11y/views/AmendReturnA11ySpec.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewSpecBase
+import forms.amends._
+import models.returns.{IdDetails, ReturnDisplayApi, ReturnDisplayDetails, TaxReturnObligation}
+import play.api.data.Form
+import play.twirl.api.Html
+import uk.gov.hmrc.scalatestaccessibilitylinter.AccessibilityMatchers
+import uk.gov.hmrc.scalatestaccessibilitylinter.domain.OutputFormat
+import viewmodels.checkAnswers.ViewReturnSummaryViewModel
+import views.html.amends._
+
+import java.time.LocalDate
+
+class AmendReturnA11ySpec extends ViewSpecBase with AccessibilityMatchers {
+
+  val aTaxObligation: TaxReturnObligation = TaxReturnObligation(LocalDate.now(), LocalDate.now().plusWeeks(12), LocalDate.now().plusWeeks(16), "PK1")
+  val amount = 321L
+
+  "AmendDirectExportPlasticPackagingView" should {
```

Comment:
I wonder if we should have a file for each view test. I just thinking about to keep the file small for readability purpose. I know the returns a11y test are all put in the same file. Maybe we should refacter them too. May we can discuss this with the rest of the team.

Reply:
Paul S had some good thoughts on this, I do think that this is neat solution to a having multiple files with identical tests run on them